Rakefile

```diff
@@ -7,4 +7,23 @@ Rake::TestTask.new(:test) do |t|
   t.test_files = FileList["test/**/*_test.rb"]
 end
 
+desc 'Import or update SCAP datastreams from the SCAP Security Guide'
+namespace :ssg do
```

Comment:
I'd like this to be added to the rails rake tasks when this gem is included. I haven't tested if that's the case already or if I need to do something special to make that happen.

Reply:
Thanks! Updated.